### ISSUE TITLE
Document builder

### ISSUE THREAD

**Issue by Libbum** [sheepsy90/urban_bootstrap#1]
Having markdown files in folders is good an all, but at some stage users will want to browse through things, search, then print hardcopies. 

Should we use something like [mkdocs](https://www.mkdocs.org/) to build up the digital side of the project?

Perhaps a more universal solution would be to use something with a [pandoc](http://pandoc.org/) backend (or just tightly integrate pandoc with the project). Then we could easily provide multiple templates to render pdf flash cards, books with indexes, whatever...

**Comment by sheepsy90** [comment]
I agree that this should be part of it. I first wanted to gather some examples to see if the categorization works out.
